Cargo.toml

```diff
@@ -79,7 +79,7 @@ self_update_crate = { version = "~0.40", default-features = false, optional = tr
 
 [target.'cfg(windows)'.dependencies]
 self_update_crate = { version = "~0.40", default-features = false, optional = true, package = "self_update", features = ["archive-zip", "compression-zip-deflate", "rustls"] }
-winapi = "~0.3"
+winapi = { version = "~0.3", features = ["consoleapi", "wincon"] }
```

Comment:
This is a completely separate change that I need to see better justification for. Please do this in a different PR.